compiler/rustc_attr_data_structures/src/attributes.rs

```diff
@@ -57,14 +57,6 @@ impl OptimizeAttr {
     }
 }
 
-#[derive(Clone, Debug, Encodable, Decodable, HashStable_Generic, PrintAttribute)]
```

Comment:
I'll add this back in soon, but that's ok

Reply:
I'd like to do that actually, I just thought there shouldn't be a bunch of dead code here at this time.

Reply:
I agree, and feel free to. I just filed #138164,  #138165 and #138291 which might help with that